Dockerfiles/Dockerfile

```diff
@@ -10,9 +10,8 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH};\
     chmod +x operator-sdk_${OS}_${ARCH};\
     mv operator-sdk_${OS}_${ARCH} operator-sdk;\
-    mv operator-sdk /usr/local/bin/;
-RUN dnf install -y git ansible wget python3-pip
```

Comment:
Please use single command per line

Reply:
Please replace `git` with `git-core` and disable weak dependencies

Lot of unnecessary weak dependencies were installed/
```
Installing weak dependencies:
 perl-IO-Socket-IP        noarch   0.41-2.fc33                  fedora     43 k
 perl-IO-Socket-SSL       noarch   2.068-3.fc33                 fedora    216 k
 perl-Mozilla-CA          noarch   20200520-3.fc33              fedora     12 k
 perl-NDBM_File           x86_64   1.15-467.fc33                updates    33 k
 python3-invoke           noarch   1.4.1-4.fc33                 fedora    149 k
 python3-paramiko         noarch   2.7.2-1.fc33                 fedora    287 k
 python3-pyasn1           noarch   0.4.8-3.fc33                 fedora    133 k
 python3-winrm            noarch   0.3.0-11.fc33                fedora     59 k

```

Reply:
This was not resolved

Reply:
according to the changelog its resolved 
@lslebodn

Reply:
@lslebodn Do you mean the weak dependencies?  I will fix it

Reply:
I request two things in the thread.

Reply:
updated to git-core and added --setopt=install_weak_deps=False